apps/web-portal/.env.defaults

```diff
@@ -1 +1,2 @@
 ZORKO_API_HTTP_PORT=7777
+ZORKO_LOGGER_PRETTY_PRINT=true
```

Comment:
why we need this logger option in configs? maybe just setup it as defaults?

Reply:
it is, however I need it here for devOps needs, to just copy and edit for example for prod or test envs